src/models/Preset.ts

```diff
@@ -44,6 +44,9 @@ export interface EnumArgs {
 
 export interface EnumPreset<R extends AbstractRenderer, O extends object = any> extends CommonPreset<R, O> {
   item?: (args: PresetArgs<R, O> & EnumArgs) => string;
+  ctor?: (args: PresetArgs<R, O> & EnumArgs) => string;
+  getValue?: (args: PresetArgs<R, O> & EnumArgs) => string;
+  fromValue?: (args: PresetArgs<R, O> & EnumArgs) => string;
```

Comment:
As these presets are used for all generators, we need to move this addition to the [JavaPreset](https://github.com/asyncapi/modelina/blob/master/src/generators/java/JavaPreset.ts) file. I think you should be able to use the following:

```ts
export interface JavaEnumPreset<O> extends EnumPreset<EnumRenderer, O> {
  ctor?: (args: PresetArgs<EnumRenderer, O> & EnumArgs) => string;
  getValue?: (args: PresetArgs<EnumRenderer, O> & EnumArgs) => string;
  fromValue?: (args: PresetArgs<EnumRenderer, O> & EnumArgs) => string;
}

export type JavaPreset<O extends object = any> = Preset<{
  class: ClassPreset<ClassRenderer, O>;
  enum: JavaEnumPreset<O>;
}>;
```